docs/repositories.md

```diff
@@ -120,53 +383,27 @@ poetry config certificates.foo.cert /path/to/ca.pem
 poetry config certificates.foo.client-cert /path/to/client.pem
 ```
 
-### Install dependencies from a private repository
+## Caches
 
-Now that you can publish to your private repository, you need to be able to
-install dependencies from it.
+Poetry employs multiple caches for package sources in order to improve user experience and avoid duplicate network
+requests.
 
-For that, you have to edit your `pyproject.toml` file, like so
+The first level cache is a [Cache-Control](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)
+cache for almost all HTTP requests.
```

Comment:
```suggestion
header for almost all HTTP requests.
```

Reply:
This should be "based cache" as we are using the cache control package with the requests session.

Reply:
Ah, then linking to docs for the header is a bit misleading.